packages/components/src/ProgressBar/ProgressBar.tsx

```diff
@@ -12,12 +13,24 @@ interface ProgressBarProps {
    * The total steps to use. For percentages you can set this to 100.
    */
   readonly totalSteps: number;
+
+  /**
+   * Changes the size to small
+   */
+  readonly size?: keyof typeof sizes;
```

Comment:
In other places in Atlantis, we've just used strings. You could do:
```suggestion
  readonly size?: "small" | "base";
```

Reply:
What @whereyoubinns has here is actually probably better for this case. It is saying that size can be any of the "classes" exported by `sizes.css`. So, if we just had the base size there we're golden!